naples/controllers/file.py

```diff
@@ -26,8 +29,10 @@ def create_file(
     content_type_override: str | None = None,
 ) -> m.File:
     try:
+        filename_without_special_characters = re.sub(RE_SPECIAL_CHARACTERS, "", file.filename if file.filename else "")
```

Comment:
how is it possible for file NOT to have filename?

Reply:
if we received file without filename, we should raise error and don't go further